app/models/katello/host/content_facet.rb

```diff
@@ -29,10 +29,26 @@ class ContentFacet < Katello::Model
       validates_with Validators::ContentViewEnvironmentValidator
 
       def update_repositories_by_paths(paths)
-        paths = paths.map { |path| path.gsub('/pulp/repos/', '') }
-        repos = Repository.where(:relative_path => paths)
+        apt_paths = []
+        yum_paths = []
+        paths = paths.reject do |path|
+          if path.starts_with? '/pulp/deb/'
+            apt_paths << path.gsub('/pulp/deb/', '')
+            true
+          elsif path.starts_with? '/pulp/repos/'
+            yum_paths << path.gsub('/pulp/repos/', '')
+            true
+          else
+            false
+          end
+        end
+        apt_repos = Repository.where(:content_type => Repository::DEB_TYPE, :relative_path => apt_paths)
```

Comment:
@jlsherrill I think this is, what breaks most of the failing tests. `content_type` has moved to the `RepositoryRoot`. Do you know, how i need to adjust that where-clause?